doc/source/package_versions.rst

```diff
@@ -1,14 +1,44 @@
+PyGranta and Granta MI compatibility
+====================================
+
+PyGranta backwards compatibility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Any reasons for tildes here when the getting_started has dashes?

Reply:
No. ReST doesn't care, but I'll standardize for consistency.